lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -40,11 +48,17 @@
   ingress:
     interface: ingress
 ```
+You _must_ register the IngressRequires class as part of the `__init__` method
+rather than, for instance, a config-changed event handler. This is because
```

Comment:
You can remove `This is because` to avoid repetition of `because`
`Doing so won't get the current relation changed event, because, ...`

Reply:
This is the ingress library, directly pulled from charmhub, so we shouldn't be touching it here